src/uu/mknod/src/mknod.rs

```diff
@@ -134,28 +185,42 @@ pub fn uu_app() -> Command {
                 .value_hint(clap::ValueHint::AnyPath),
         )
         .arg(
-            Arg::new("type")
+            Arg::new(options::TYPE)
                 .value_name("TYPE")
                 .help("type of the new file (b, c, u or p)")
                 .required(true)
                 .value_parser(parse_type),
         )
         .arg(
-            Arg::new("major")
-                .value_name("MAJOR")
+            Arg::new(options::MAJOR)
+                .value_name(options::MAJOR)
                 .help("major file type")
                 .value_parser(value_parser!(u64)),
         )
         .arg(
-            Arg::new("minor")
-                .value_name("MINOR")
+            Arg::new(options::MINOR)
+                .value_name(options::MINOR)
                 .help("minor file type")
                 .value_parser(value_parser!(u64)),
         )
+        .arg(
+            Arg::new(options::SELINUX)
+                .short('Z')
+                .help("set SELinux security context of each created directory to the default type")
+                .action(ArgAction::SetTrue),
+        )
+        .arg(
+            Arg::new(options::CONTEXT)
+                .long(options::CONTEXT)
+                .value_name("CTX")
+                .value_parser(value_parser!(String))
+                .num_args(0..=1)
```

Comment:
You also have to specify `require_equals`, otherwise something like `cargo run --features=unix,feat_selinux mknod --context hello_world p` will fail.
```suggestion
                .num_args(0..=1)
                .require_equals(true)
```